src/chrome/content/rules/communities.gov.uk.xml

```diff
@@ -32,6 +32,8 @@
 		Status code mismatch:
 		- mcis.erdf.communities.gov.uk
 		- reporting.erdf.communities.gov.uk
+		- maps.communities.gov.uk
+			<test url="http://maps.communities.gov.uk/geoserver/dclg_inspire/ows?service=WFS&version=2.0.0&request=GetFeature&typeName=dclg_inspire:Local_Authority_Greenbelt_boundaries_2017-18&outputFormat=shape-zip&srsName=EPSG:4326" />
```

Comment:
not sure about this line

Reply:
It's the example of a page where you can see that the domain is broken. What would you rather have me do?

Reply:
`Example URL: http://maps.communities.gov.uk/geoserver/dclg_inspire/ows?service=WFS&version=2.0.0&request=GetFeature&typeName=dclg_inspire:Local_Authority_Greenbelt_boundaries_2017-18&outputFormat=shape-zip&srsName=EPSG:4326` would make more sense.